sources/packages/web/src/assets/css/base.scss

```diff
@@ -112,3 +112,21 @@ ul.no-bullets {
 .p-tabview-panels {
   margin-top: 16px;
 }
+
+.form-header-title {  
```

Comment:
Are these css classes written to be used in form.io? If so please move these classes to form-shared.scss

Reply:
In the context of this ticket, they are only applicable to forms created in vue. I'm not sure if they will be used in form.io in future.

Reply:
mm, can we rename the class without `form`. maybe something like `header-title`

Reply:
Yeah exactly that was my point